style.css

```diff
@@ -0,0 +1,69 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap");
+
+body {
+  font-family: "Poppins", sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+nav {
+  height: 48px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+  color: #6070ff;
+  background-color: #fff;
+}
+
+.menu {
+  display: flex;
+  flex-direction: column;
+  width: 24px;
+  padding: 4px;
+  cursor: pointer;
+}
+
+.menu span {
+  display: block;
+  height: 2px;
+  background-color: #6070ff;
+}
+
+.menu span:not(:last-child) {
+  margin-bottom: 5px;
+}
+
+.hero {
+  padding: 140px 24px;
```

Comment:
- [ ] [OPTIONAL] I think the top padding of the hero section is too much as it is causing your top main title to misalign with the design.
 I believe you can reduce the padding pixel value.

Your Design                                                        |  Figma Design                                |
:-------------------------------------------------------------:|:-----------------------------------------------:
![Your Design](https://user-images.githubusercontent.com/39523312/143264276-8f17913a-5a47-46bf-9d24-dcd00bc01912.png)   |  ![Figma Design](https://user-images.githubusercontent.com/39523312/143264329-ca305dd0-0a34-41fe-8652-27ed08091a23.png)|